test/extensions/transport_sockets/http_11_proxy/connect_integration_test.cc

```diff
@@ -450,7 +452,8 @@ TEST_P(Http11ConnectHttpIntegrationTest, DfpNoProxyAddress) {
   EXPECT_THAT(waitForAccessLog(access_log_name_), testing::HasSubstr("dns_resolution_failure"));
 }
 
-TEST_P(Http11ConnectHttpIntegrationTest, DfpWithProxyAddress) {
+// Regression tests https://github.com/envoyproxy/envoy/issues/34096
+TEST_P(Http11ConnectHttpIntegrationTest, DfpWithProxyAddressAndContentLength) {
```

Comment:
Should the test *without* content length also still exist?

Reply:
Didn't get a response on this comment. Seems like both paths should be getting tested.

Reply:
yeah sorry we do that by default - all th eother tests test this with  stripConnectUpgradeAndRespond()